plugins/webp-uploads/hooks.php

```diff
@@ -780,3 +780,31 @@ function webp_uploads_init(): void {
 	}
 }
 add_action( 'init', 'webp_uploads_init' );
+
+/**
+ * Automatically opt into extra image sizes when generating fallback images.
+ *
+ * @since n.e.x.t
+ */
+function webp_uploads_opt_in_extra_image_sizes(): void {
+	if ( ! webp_uploads_is_fallback_enabled() ) {
+		return;
+	}
+
+	global $_wp_additional_image_sizes;
+
+	if ( ! is_array( $_wp_additional_image_sizes ) ) {
+		return;
+	}
```

Comment:
```suggestion
```

Don't think we needs to add these check. We already added `isset` check so it will not return any error.